### DISCUSSION TITLE
Differences between marijuana use in "su_y_sui" and "su_y_mypi"

### DISCUSSION THREAD

**Discussion by john-jardine** [now-i-know-my-abcd/docs#84] [Stats & Code Help]
Hello,

I have been working with the full-year substance use interview dataset (su_y_sui) and the mid-year substance use phone interview dataset (su_y_mypi).

In the codebook for "su_y_sui", there is a variable "tlfb_mj_use" with the description, "I want to start by asking if you have EVER TRIED any of the following drugs in your life. Have you ever tried___________at any time in your life? Smoked marijuana, also called pot, grass, weed, ganja - more than just a puff?"

In the codebook for "su_y_mypi", I think the variable closest to replicating this measure is "mypi_mj_used", with the description "Have you smoked marijuana flower or bud in a pipe, joint, hookah or bong at any time in the past 6 months?"

Do you think it's fair to assume these measures are _not_ equivalent, given that "mypi_mj_used" does not specify "more than just a puff." I am trying to leverage all the information available from both of these datasets, but this seems like an incongruity to me.

There is another variable "mypi_mj_30" with the description, "Have you had a puff or vape of any marijuana, cannabis or 'weed,' or tried drinks or food that have marijuana in them in the past 6 months?" However, this variable is only available for the 30-month interview and later. And it also combines puffing marijuana with edibles / drinks.

I am wondering if anyone has more information about if the question wording is exactly as it appears in the codebook, or if you have any experience combining the information from these datasets. I have seen many records in the full-year dataset where respondents indicate not using a substance, but the same respondents indicate using that substance during the corresponding mid-year phone interview, which is why I want to leverage all information possible.

Thanks.

**Comment by swhawes** [comment]
lmk if the following info helps 👍 –– _It looks like at the 30mo mid-year phone interview the script changed to asking about a puff and more than a puff, but before that it just asked if they had used marijuana._
